its/plugin/it-python-plugin-test/src/test/java/com/sonar/python/it/plugin/IPythonTest.java

```diff
@@ -84,50 +85,52 @@ public static void stop() {
   }
 
   @Test
-  public void should_raise_issues() throws IOException {
-    ClientInputFile inputFile = prepareInputFile("foo.ipynb",
-      "def fooBar():\n"
-        + "  `1` \n"
-        + "  `1` #NOSONAR\n",
-      false);
-
-    List<Issue> issues = new ArrayList<>();
-    StandaloneAnalysisConfiguration configuration =
-      StandaloneAnalysisConfiguration.builder()
-        .setBaseDir(baseDir.toPath())
-        .addInputFile(inputFile)
-        .setModuleKey("myModule")
-        .build();
-
-    Map<ClientLogOutput.Level, List<String>> logsByLevel = new HashMap<>();
-    ClientLogOutput logOutput = (s, level) -> {
-      List<String> logs = logsByLevel.getOrDefault(level, new ArrayList<>());
-      logs.add(s);
-      logsByLevel.putIfAbsent(level, logs);
-    };
-    ClientModuleFileSystem clientFileSystem = new ClientModuleFileSystem() {
+  public void shouldRaiseIssues() throws IOException {
+    var inputFile = createInputFile(Path.of("projects/ipynb_project/file1.ipynb"), "file1.ipynb", false);
```

Comment:
Not asking for change because this is personal preference, but I sometimes wonder what is the value of using `var`. In IntelliJ, with the type inference tooltip, the resulting line is longer (because we have  both`var` AND the name of the type) and (I find) less readable than by just declaring the variable as `ClientInputFile`.

This is not always the case (for example, the `HashMap` is indeed less verbose), but sometimes I find it doesn't bring that much value and that we maybe shouldn't use it systematically (I admit I feel this is an unpopular opinion though 😄). What's your opinion on this?

Reply:
discussed